### ISSUE TITLE
WiFi wpa_supplicant failed / no link

### ISSUE THREAD

**Issue by ueen** [avanc/photOS#128]
After some trouble we got a USB WiFi stick to show as wlan0 and we also get a connection manually setting iwconfig but after a reboot all fail author l without an ethernet connection. 
Please help, wpa_supplicant.conf is no folder or do we need further settings to somehow connect it to wlan0?

**Comment by avanc** [comment]
So you got the stick working manually? In that case, the needed kernel modules are available. Might it be that you loaded the drivers manually (e.g. modprobe)? Maybe they have to be defined to be loaded on startup.

**Comment by ueen** [comment]
Not really, it worked with lan and the stick and it worked when removing Lan (it stayed on).
But booting only with stick never worked. 
I loaded no drivers, as I found none, should be plug n play no?

**Comment by avanc** [comment]
@ueen Are you able to connect to the device via Wifi when you disconnected the LAN? Or is the device just not rebooting?

**Comment by ueen** [comment]
Right, with the stick and Lan it works when I remove the Lan (no reboot) whereas a reboot occurs if there's only Lan and I remove it. So my conclusion would be that there is a wlan connection. But it only works when there's a lan connection first.

**Comment by avanc** [comment]
When both is working (LAN+WIFI) youl coud to "lsmod" to see if any specific wifi module is loaded. If there are specific modules it might be an option to configure them to be loaded on startup.

**Comment by ueen** [comment]
Hmm do you have a guide on how to do it? 
Also I noticed the txt withel the WiFi details is always disappearing from the SD card after a reboot.
